02week/bio/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Katie Macaulay Bio</title>
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display" rel="stylesheet">
+    <link rel="stylesheet" href="css/reset.css"/>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <div class="into">
+      <header>
+        Katie Macaulay
+      </header>
+      <h1>Web Developer, Creative</h1>
+    </div>
+    <main>
+        <aside>
+            <ol class=links>
+              <li>
+                <a href="/Users/katiemacaulay/Desktop/web-workbook/02week/blog/index.html" target="_blank">
```

Comment:
The link here does not need to go all the way back to your users folder locally. Your link should look more like: href="./blog/index.html" 
(you are in your index in the bio right now, so you take a step back by ./ then you say find blog folder with blog/ then in there, open the index.html.)